portfolio/src/main/webapp/script.js

```diff
@@ -84,7 +84,7 @@ function createPTag(text) {
 
 /**
  * Reveals hidden description upon mouseover and hides the project title
- *  text.
+ *     text.
```

Comment:
I am confused as to why there is extra spacing. Is this something asked for in the style guide?

Reply:
I think I got carried away with the spacing. Thank you for catching that! I think there should only be spacing for the tags. https://google.github.io/styleguide/jsguide.html#jsdoc-line-wrapping

Done!